src/sentry/static/sentry/app/utils.jsx

```diff
@@ -224,8 +224,6 @@ export function sortProjects(projects) {
 export const buildUserId = id => `user:${id}`;
 export const buildTeamId = id => `team:${id}`;
 
-export const actorEquality = (a, b) => a.type === b.type && a.id === b.id;
```

Comment:
maybe leave this in? we might want it later. nbd tho

Reply:
I don't think we'll need it